SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1283,13 +1286,18 @@ void USpatialNetDriver::ProcessOwnershipChanges()
 	{
 		if (USpatialActorChannel* Channel = GetActorChannelByEntityId(EntityId))
 		{
+			check(IsValid(Channel->Actor));
```

Comment:
Perhaps consider ensuremsg rather than assert?

Reply:
This is an invariant here as entity IDs are only added to `OwnershipChangedEntities` when they have a channel assigned. Changing this to an `ensure` might be better aligned with what we've discussed relating to stability - let's pick this up in an offline discussion.

Reply:
Not sure where we landed, but ensuremsg seems safer from my perspective.